tauri/src/types/alchemy_response.rs

```diff
@@ -0,0 +1,16 @@
+use ethers_core::types::{Address, U256};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AlchemyResponse {
+    pub address: Address,
+    pub token_balances: Vec<TokenBalance>,
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TokenBalance {
+    pub contract_address: Address,
+    pub token_balance: U256,
+}
```

Comment:
I think this new type should live on the same module as the listener, since it seems to be a more scoped thing
this types module is intended for broad things (e.g.: types that will be used in many places, or that need to be replicated on the UI as well)